### ISSUE TITLE
Remove prefix `yii-` from `yii-bootstrap5`

### ISSUE THREAD

**Issue by Tigrov** [yiisoft/bootstrap5#129]

**Comment by vjik** [comment]
If we do it, then need make optional dependencies for Yii Assets and etc. Also need improve readme.

**Comment by samdark** [comment]
I don't think it makes sense for this package.

**Comment by vjik** [comment]
By fact, this package contain widgets, that generate HTML for Bootstrap 5. With this positioning prefix "yii-" not need.

**Comment by luizcmarin** [comment]
the other packages do not use the "yii-" prefix. Why make an exception to the general pattern?

**Comment by vjik** [comment]
> the other packages do not use the "yii-" prefix. Why make an exception to the general pattern?

Some packages are used `yii-prefix`. See docs: https://github.com/yiisoft/docs/blob/master/000-packages.md#yii-specific-packages-framework-and-extensions

**Comment by luizcmarin** [comment]
so it should at least be yii1 yii2 or yii3, because, for me at least, it creates a lot of confusion and headaches having to figure out who is what, including because the yii3 documentation for bootstrap5 is a copy of 4, which makes me wonder that is out of date or the package has not changed at all.
but look at the image and imagine being new or a developer returning after a long time. see the first and last package in the image. Do you realize that we need to look for which version each package belongs to and, in this case, you need to look inside the folder for something that identifies it?
I just want to say that there are simple things that make things a lot easier. And I will read all yii3 packages and report everything that can be improved. And I don't even know Yii... but I believe in it and that it will be released in November this year. and I really want to convert my application to it. That's why I'm trying hard to learn.
why not name everything from version 3 "yii3-"? for version 2 they did this (look at the image), Just a humble suggestion.
Sorry for interfering in the matter without being invited.
![Screenshot_2024-04-07-19-50-36-643_com android chrome](https://github.com/yiisoft/yii-bootstrap5/assets/67489841/dba79a19-14e6-43ea-b26f-371ac3879e56)

**Comment by terabytesoftw** [comment]
It would be great to be able to define it, if we rename it to `yii3-bootstrap5` it makes sense, or just `bootstrap5`, but in general terms the package depends on other `yii` packages, so i would vote for `yii3-bootstrap5`.

**Comment by samdark** [comment]
@terabytesoftw no other Yii3 packages follow `yii3-*` naming, so either we should adjust them all or use naming that's similar to the ones already released. I'd vote for not changing existing released package names.

**Comment by vjik** [comment]
Yes, we have conflict in names with Yii1 packages: both packages use names "yii-". But all Yii1 packages with "yii-" names is abandoned and marked as read only.

So, I suggest keep [our convention](https://github.com/yiisoft/docs/blob/master/000-packages.md#yii-specific-packages-framework-and-extensions) and use "yii-" prefix for Yii3 specific packages.

But as for this package, I think we should rename it to "yiisoft/bootstrap5", because this package provides widgets, that generate HTML for Bootstrap 5. It's not Yii3 specific.

**Comment by samdark** [comment]
Makes sense. Let's do it.
